Scala Survey 2022 Announcement (#1420)
scala/scala-lang: blog/_posts/2022-09-29-scala-survey-2022-announcement.md

---
layout: blog-detail
post-type: blog
by: Kacper Korban, VirtusLab
title: Participate in the Scala Survey 2022!
---

![Scala Survey 2022 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2022-announcement.png)

## The Scala Team wants to hear from you!

> We want to make the language we all love as amazing as possible. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why ask you to help by participating in the Scala Survey 2022.

Reviewer: "that's why _we_ want to..."

Maybe consider a more open style "we invite you to..."

> We want to make the language we all love as amazing as it possibly can be. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why we are asking you to help by taking part in the Scala Survey 2022.

Reviewer 2: How about making `Scala Survey 2022` here a link instead of adding it below?

Author: I thought about it, but the whole point of this post is to give the link to the survey. And this way it is more explicit and more visible.

> We want to make the language we all love as amazing as it possibly can be. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why we are asking you to help by taking part in the Scala Survey 2022.

Reviewer 3: ```suggestion
We want to make the language we all love as amazing as possible. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why ask you to help by participating in the Scala Survey 2022.
```

> Survey: [Scala Survey 2022](https://scala-survey-2022.virtuslab.com/)

Reviewer: The repetition of survey looks bad. Consider writing just "Scala survey 2022“

## About the survey

> Completing the survey should take 4-12 minutes. The survey is anonymous and most of the questions are optional, so if you don’t have that much time, you don’t have to answer all of them.

Reviewer 3: ```suggestion
Completing the survey should take 4-12 minutes. The survey is anonymous, and most of the questions are optional, so if you don’t have that much time, you don’t have to answer all of them.
```

Completing the survey should take 4-12 minutes. The survey is anonymous, and most of the questions are optional, so if you don’t have that much time, you don’t have to answer all of them.

**Who is the survey intended for?** Everyone. Whether you’re an experienced Scala Developer or just starting your Scala journey, we count on your participation.

> After the survey closes, we will publish the aggregated results on the [Scala blog](https://www.scala-lang.org/blog/).

Reviewer 4: Could we have a line here about the content of the survey's questions, i.e overview of the topics covered

Reviewer 4: e.g. We are interested in learning what libraries and tools people are using, what platforms they support, the future direction of tooling focus, where you are learning to use scala/training engineers, etc.

> Spread the word! Help us get answers from as many diverse groups as possible. So share the survey on social media, your office, your university, meetups, and other communities.

Reviewer 2: > So share the survey on social media, your office, your university, meetups, and other communities

I guess grammatically other prepositions should be added, like `in your office, at your university, ...` or we could somehow reformulate this sentence to remove the need for prepositions

> The Survey will close on **21.10.2022**.

Reviewer: This should probably go earlier (before line 22)

## The Scala Team wants to hear from you!

We want to make the language we all love as amazing as possible. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why we invite you to help by participating in the Scala Survey 2022.

[Scala Survey 2022](https://scala-survey-2022.virtuslab.com/)

The Survey will close on **21.10.2022**.

## About the survey

> 

Reviewer 4: The next line needs to be more persuasive for users to complete the survey, this should just say how long the survey takes

Reviewer 4: can keep the part about it being anonymous also
